packages/discord.js/src/structures/BaseGuildVoiceChannel.js

```diff
@@ -91,8 +91,8 @@ class BaseGuildVoiceChannel extends GuildChannel {
    * // Remove a fixed region for this channel - let Discord decide automatically
    * channel.setRTCRegion(null);
    */
-  setRTCRegion(region) {
-    return this.edit({ rtcRegion: region });
+  setRTCRegion(rtcRegion) {
```

Comment:
You must also rename the parameter in `index.d.ts`

https://github.com/discordjs/discord.js/blob/d32db8833e1058fb36f2e83af79d5353a9f2f693/packages/discord.js/typings/index.d.ts#L434

Reply:
Need to change the parameter name in the jsdoc above the method